cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/network/netty/NettyPacketLengthSerializer.java

```diff
@@ -12,10 +12,11 @@
 
     @Override
     protected void encode(ChannelHandlerContext ctx, ByteBuf in, ByteBuf out) {
-        int readableBytes = in.readableBytes(), lengthByteSpace = getVarIntSize(readableBytes);
+        int readableBytes = in.readableBytes();
+        int lengthByteSpace = getVarIntSize(readableBytes);
 
-        if (lengthByteSpace > 5) {
-            throw new IllegalArgumentException();
+        if (lengthByteSpace > 3) {
```

Comment:
Why did you change this from 5 to 3? Some packets are bigger and can't be sent with this